MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2368,18 +2352,10 @@ public void TryStreamCopy(EncodingJobInfo state)
             }
         }
 
-        public string GetProbeSizeArgument(int numInputFiles)
```

Comment:
Why remove these `FFmpeg:probesize` and `FFmpeg:analyzeduration`  environment variables? 
Relates to https://github.com/jellyfin/jellyfin/issues/3930

Reply:
After this change the `numInputFiles` becomes always 1, thus ther result of the method will be always `string.Empty`